products/idn/api/authorization.md

```diff
@@ -41,35 +41,42 @@ sequenceDiagram
+
+User levels are typically granted through the UI, [following the procedures from this document](https://documentation.sailpoint.com/saas/help/common/users/grant_remove_user_levels.html).
 
 :::caution
 
 There is an [API that can set an identity's user level](https://developer.sailpoint.com/discuss/t/assign-identitynow-admin-roles-via-api/1874/4), but it is a V1 API with no guaranteed support. Use it at your own risk!
 
 :::
 
-You can use user levels to limit access to API integrations, but their rigidity and broad scopes makes them difficult to configure for integrations that may only need access to a few endpoints. The next section covers scopes, which you can use to refine and limit an API integration's access at a more granular level than you can with user levels.
+User levels act as the first line of defense by applying a rigid boundary around the APIs that a user can call.  The next section introduces scopes, which allow users to apply granular controls on the APIs an access token can call.
 
 ## Scopes
 
-Scopes are granular permissions you can add to personal access tokens (PATs) to create tokens with the least privilege necessary to fulfill their functions. Unlike user levels, which apply to any PATs created by a user, scopes can be unique to each PAT. This allows a single user to have multiple credentials with different privileges that support unique use cases and software applications. Using scopes is beneficial to security - if a bad actor compromises any one of the credentials, the bad actor can only perform the limited set of operations defined by the credential's scopes, significantly reducing the potential damage that can be done.
+Scopes are granular permissions you can add to personal access tokens (PATs) to create tokens with the least privilege necessary to fulfill their functions. User levels place a broad border around the APIs a token has access to, while scopes allow each token to specify which endpoints within the user level it needs.  Scopes allow an API user to have multiple tokens with different privileges that support unique use cases and software applications. Using scopes is beneficial to security - if a bad actor compromises any one of the tokens, the bad actor can only perform the limited set of operations defined by the token's scopes, significantly reducing the potential damage that can be done.  Therefore, it is recommended that all users apply scopes to each PAT they create in order to reduce the impact of stolen credentials.
```

Comment:
I think we should revisit this: "while scopes allow each token to specify which endpoints within the user level it needs."  I _think_ that scopes don't have to be a 1-1 correspondence with endpoints, so the way I see scopes, they _allow you to limit the set of endpoints that token will be allowed to invoke_.

The rest of this paragraph is spot on.

Reply:
I updated the wording to incorporate your feedback.